python/robot-framework/requirements.txt

```diff
@@ -0,0 +1,3 @@
+robotframework==7.0.0
+robotframework-seleniumlibrary==6.2.0
```

Comment:
Are those a requirement ? Since this is a dependency for `saucelabs_visual` shouldn't they get pulled automatically ?

Reply:
I don't have them as dependencies since they're only used when someone specifically uses the robot framework package from this -- otherwise it's just bloat I think? Maybe we can do named dependencies but I didn't know how to do it for the first release.

Reply:
Let's keep it that way. We would probably expect the developer to already have those in their projects.